libtransmission/announcer.c

```diff
@@ -1303,6 +1357,24 @@ static void tierAnnounce(tr_announcer* announcer, tr_tier* tier)
 ****
 ***/
 
+static bool multiscrape_too_big(char const* errmsg)
+{ 
+  if (!errmsg)
+      return false;
+
+  if (strstr(errmsg, "GET string too long"))
```

Comment:
Do we want to create a `NULL`-terminated array and loop over it here? More complex cases will still have to be handled separately, but there're no such cases now and 3 `if`-s in a row look like a pattern to me.

Reply:
Added in fab60721d0a3c1f7f118c6e0d2c4d581867d8e10